internal/pkg/fixtures/fixtures.go

```diff
@@ -92,6 +92,7 @@ type StateFile struct {
 	AllBranchesConfigs []string          `json:"allBranchesConfigs" validate:"required"`
 	Branches           []*BranchState    `json:"branches" validate:"required"`
 	Buckets            []*Bucket         `json:"buckets,omitempty"`
+	Sandboxes          []*Sandbox        `json:"sandboxes,omitempty"`
```

Comment:
Bolo by zlozite hodit `sandboxes` priamo pod `Branch`, ak som pozeral spravne, tak v API podporujeme `brancheId`.
Nie je to must have, ale ak je to jednoduche, tak by som preferoval spravit to kompletne.

Reply:
Složité by to nebylo. V API to podporujeme, ale v CLI máme vždy default branch

Reply:
Tak by si to tu upravil?, v CLI to nebude problem pridat, ... ale nechces potom asi prepisovat zbytocne existujuce testy.
Pripadne pridat rovno aj do CLI? (alebo zalozit task do dalsieho sprintu), ... neviem odhadnut konkretne kolko to je casu, davam approve, a nechcem to na teba.

Reply:
Jo, dodělat to v těch commandech atd. by trvalo mnohem déle. Taky DBT commandy by se potom měly upravit, taky berou vždy jen default branch. To je teda spíš na další sprint, vytvořím placeholder task

Reply:
Mám ten task dát do stejné iniciativy?

Reply:
dbt by som nechal ako je, to ma svoj usecases, workspaces by mali byt univerzalne, ok :wink:

Reply:
@jprochazk nie, lebo ju nezavrieme :slightly_smiling_face:  do maintenance (mozes vytvorit epic na maintenance pre dalsi sprint, podobne, ako mame v tomto)

Reply:
ono cela ta podpora workspaces je nasa iniciativa (ked uz sme to implementovali do API, tak sme chceli nech je to aj v CLI).

Reply:
https://keboola.atlassian.net/browse/KAC-273